opengrok-web/src/main/webapp/foot.jspf

```diff
@@ -42,10 +42,12 @@ org.opengrok.indexer.web.Prefix"
     
         %>
     <div id="footer">
-<p><a href="https://oracle.github.io/opengrok/"
- title="Served by OpenGrok (<%= Info.getVersion() %> - <%= Info.getShortRevision() %>)"><span id="fti"></span></a></p>
+<p><a href="https://oracle.github.io/opengrok/" class="link-no-decoration"
+      title="Served by OpenGrok (<%= Info.getVersion() %> - <%= Info.getShortRevision() %>)">
+    <span id="served-by">served&nbsp;by&nbsp;</span><span class="logo logo-gray-color">{</span><span class="logo logo-blue-color">Open</span><span class="logo logo-gray-color">Grok</span>
```

Comment:
The spans need to be on the same line due to https://stackoverflow.com/questions/2519923/get-rid-of-spaces-between-spans -> I did not like the other solutions as they are mostly hacks.